src/Form/_index.scss

```
@@ -434,6 +433,8 @@ select.form-control {
 }
 
 .pgn__form-checkbox-input {
+  border-radius: var(--pgn-size-form-control-border-checkbox-indicator-radius) radius;
```

Thanks for moving this `border-radius` property!

One last thing I think we should update before merging is removing the trailing `radius` in this line, as I don't believe it's technically valid CSS syntax for `border-radius` ([spec](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius)).

```suggestion
  border-radius: var(--pgn-size-form-control-border-checkbox-indicator-radius);
```

---

Sorry, I fixed it.